LoopbackApp/app/src/main/cpp/Android.mk

```diff
@@ -18,20 +18,15 @@ LOCAL_SRC_FILES   := \
     lb2/loopback2.cpp \
     lb2/sound_system_aaudio.cpp \
     lb2/oboe/src/aaudio/AAudioLoader.cpp
+
 LOCAL_C_INCLUDES := \
```

Comment:
isn't this for ndk-build or Android platform build?  If so, then it shouldn't need changes

Reply:
this one is needed if someone wants to use gradle + android.mk combination. I tested it with gradle it works. probably could add into the readme to say: if people wants to use Android.mk + Gradle, it is ready to use